### PR TITLE
[INLONG-5562][Manager] Add the workflow API in the manager client

### LINKED ISSUE #5562: [Feature][Manager] Supplement the workflow API in the manager client
### Description

Supplement the workflow API in the manager client

### Use case

Supplement the workflow API in the manager client

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
- Fixes #5562 

### Motivation

Supplement the API that the manager web exists but the client does not exist for manager client.

### Modifications

Add the workflow API in the manager client

### Verifying this change

- [x] This change added tests and can be verified as follows:
  - *org.apache.inlong.manager.client.api.inner.WorkflowClientTest#testWorkflowStart*
  - *org.apache.inlong.manager.client.api.inner.WorkflowClientTest#testListProcess*

### CODE REVIEW

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/WorkflowApi.java [hunk 1]
```diff
@@ -37,8 +43,29 @@ public interface WorkflowApi {
...
+    @POST("workflow/reject/{taskId}")
+    Call<Response<WorkflowResult>> reject(@Path("taskId") Integer taskId, @Body WorkflowOperationRequest request);
+
+    @POST("workflow/complete/{taskId}")
+    Call<Response<WorkflowResult>> complete(@Path("taskId") Integer taskId, @Body WorkflowOperationRequest request);
+
+    @GET("workflow/detail/{id}")
+    Call<Response<ProcessDetailResponse>> detail(@Path("processId") Integer processId, @Query("taskId") Integer taskId);
+
+    @GET("workflow/listProcess")
```
Comment (leosanqing) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/WorkflowApi.java:65]: I remember Get doesn't use objects, please confirm. If not, you need to use Map to receive QueryParam

You can refer this ISSUE https://github.com/square/retrofit/issues/2293#issuecomment-683286832

https://github.com/square/retrofit/issues/2293#issuecomment-683286832

Comment (haifxu) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/WorkflowApi.java:65]: Yeah, the [user API](https://github.com/apache/inlong/pull/5494/files) has the same problem, I will fix it later.

File: inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/WorkflowClientTest.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                        )
+        );
+        WorkflowOperationRequest request = new WorkflowOperationRequest();
+        request.setName(ProcessName.APPLY_GROUP_PROCESS);
+        request.setApplicant("test_user");
+        ApplyGroupProcessForm form = new ApplyGroupProcessForm();
+        form.setGroupInfo(new InlongPulsarInfo());
+        request.setForm(form);
+
+        WorkflowResult workflowInfo = workflowClient.start(request);
```
Comment (leosanqing) [inline inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/WorkflowClientTest.java:71]: The order of the two is reversed, the first should be the expected value and the second is the actual value

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/InlongStreamApi.java [hunk 3]
```diff
@@ -45,24 +45,24 @@ public interface InlongStreamApi {
...
             @Query("streamId") String streamId);
 
-    @POST("/stream/list")
+    @POST("stream/list")
     Call<Response<PageInfo<InlongStreamBriefInfo>>> listByCondition(@Body InlongStreamPageRequest request);
 
     @POST("stream/listAll")
     Call<Response<PageInfo<InlongStreamInfo>>> listStream(@Body InlongStreamPageRequest request);
 
-    @POST("/stream/startProcess/{groupId}/{streamId}")
```
Comment (leosanqing) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/InlongStreamApi.java:54]: These have nothing to do with this ISSUE, mention ISSUE and PR separately

File: inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/WorkflowClientTest.java (hunk 2 above)
Comment (haifxu) [reply inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/WorkflowClientTest.java:71]: Done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/InlongStreamApi.java (hunk 3 above)
Comment (haifxu) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/InlongStreamApi.java]: OK , I have reset and then fix it in next PR.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/WorkflowApi.java [hunk 4]
```diff
@@ -37,8 +40,29 @@ public interface WorkflowApi {
...
     Call<Response<WorkflowResult>> startInlongGroup(@Path("taskId") Integer taskId, @Body Map<String, Object> request);
 
-    @GET("workflow/event/list")
-    Call<Response<PageInfo<EventLogResponse>>> getInlongGroupError(@Query("inlongGroupId") String groupId,
-            @Query("status") Integer status);
+    @POST("workflow/start")
+    Call<Response<WorkflowResult>> start(@Body WorkflowOperationRequest request);
+
+    @POST("workflow/cancel/{taskId}")
+    Call<Response<WorkflowResult>> cancel(@Path("processId") Integer processId, @Body WorkflowOperationRequest request);
```
Comment (leosanqing) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/WorkflowApi.java:47]: Is this "taskId"?

Comment (haifxu) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/WorkflowApi.java:47]: Done

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Review by leosanqing — APPROVED** [review]

**Pull request merged**